cylc/flow/scripts/clean.py

```diff
@@ -18,39 +18,50 @@
 
 """cylc clean [OPTIONS] ARGS
 
-Remove a stopped workflow from the local scheduler filesystem.
+Remove a stopped workflow from the local scheduler filesystem and remote hosts.
 
 NOTE: this command is intended for workflows installed with `cylc install`. If
 this is run for a workflow that was instead written directly in ~/cylc-run and
 not backed up elsewhere, it will be lost.
 
-It will also remove an symlink directory targets. For now, it will fail to
-remove workflow files/directories on a remote host.
+It will also remove any symlink directory targets.
 
 Suite names can be hierarchical, corresponding to the path under ~/cylc-run.
 
 Examples:
-  # Remove the workflow at ~/cylc-run/foo
-  $ cylc clean foo
+  # Remove the workflow at ~/cylc-run/foo/bar
+  $ cylc clean foo/bar
 
 """
 
 from cylc.flow.option_parsers import CylcOptionParser as COP
 from cylc.flow.terminal import cli_function
-from cylc.flow.suite_files import clean
+from cylc.flow.suite_files import clean, init_clean
 
 
 def get_option_parser():
     parser = COP(
         __doc__,
-        argdoc=[("REG", "Suite name")]
+        argdoc=[('REG', "Suite name")]
     )
+
+    parser.add_option(
+        '--local-only', metavar='RUN_DIR',
+        help="(For internal use) Path of the workflow run dir; will only "
+             "clean on local filesystem.",
+        action='store', dest='local_only__run_dir'
+    )
```

Comment:
Why do we need to pass the path, can't that be determined from the REG?

When we add more functionality this option will make sense for other purposes (we will want a `--remote-only` option to go with it) so can remove the (internal only) flag.

```suggestion
    parser.add_option(
        '--local-only', '--local',
        help="Only clean on local filesystem.",
        action='store_true',
        default=False
    )
```